cluster-appliances/configure

```diff
@@ -39,29 +39,53 @@ write_file('cluster-appliances.rc', appliances_vars.join("\n"))
 RUBY
 }
 
+_enable_alces_flight_www() {
+    local role, certname
+    role="$1"
+    files_load_config config config/cluster
+    
+    certname = "${role}.${cw_CLUSTER_name}"
```

Comment:
Pedantry again. Having spaces around `=` in an assignment isn't idiomatic (or at least, isn't the style used elsewhere).  Could we lose them please?